src/main/java/org/jabref/gui/collab/ExternalChangesResolverViewModel.java

```diff
@@ -20,6 +20,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.jabref.gui.Globals.stateManager;
```

Comment:
Globals is deprecated and shall not be used. Please inject the statemanager in the constructor.